main/acle_pdf_conf.yaml

```diff
@@ -0,0 +1,15 @@
+title: Arm C Language Extensions
+version: 2021Q2
+date: 02 July 2021
+logo: ./tools/Arm-logo-blue-RGB.pdf
+documentclass: article
+toc: true
+numbersections: true
+geometry: landscape,top=3.7cm,bottom=2.7cm,left=3cm,right=2cm,headsep=1.5cm,footskip=.5cm
```

Comment:
I still think we have to get rid of the landscape layout. I have seen the screenshot you have posted showing the issue with the table ( https://github.com/ARM-software/acle/pull/101#discussion_r743821880). I think we can fix the issue as follows:

1. Convert the content of the `Macro name` columns into hyperlinks to the link specified in the columns `See section`. This will create a table with only two fields that should print correctly.
2. This solution would make the links invisible when printing the PDF on paper. Therefore, we need to find a way to render the hyperlinks in textx that wil be printed. The option `links-as-notes` of https://pandoc.org/MANUAL.html#variables-for-latex seems to be what we need. 


May I ask you to post a screenshot of the page of the table once you have given it a go with 1 and 2?

Reply:
I can't quite understand the idea. Create footnotes for the tables, and put them where? In "See section"?

Reply:
Me and Valeria have clarified what needs a to be done here. A separate PR will be created to fix the rendering of the tables.

Reply:
The margin values are based on what I measured from the example document, which was in portrait format.

Reply:
oh, I see, I thought they were specific to the landscape layout! Never mind them, just remove the `landscape` option.